app/assets/stylesheets/base/general.scss

```diff
@@ -1,3 +1,7 @@
+script {
+  display: none !important;
+}
+
```

Comment:
There is the rule in css on some pages:
```
... > * {
  display: flex
}
```
it caused displaying script tag with all contents.

Reply:
Thanks for the clarification. Yeah, not sure why such a open css rule was initially added but we'll definitely try and remove it in a later refactor. The `display: none !important` fix is fine as a workaround for now.